tests/suite/resources_utils.py

```diff
@@ -1237,4 +1250,24 @@ def get_service_endpoint(kube_apis, service_name, namespace):
             if ex.status == 500:
                 print("Reason: Internal server error and Request timed out")
                 raise ApiException
-    return ep
\ No newline at end of file
+    return ep
+
+def get_last_reload_time(req_url, ingress_class) -> str:
+    
+    reload_metric = ""
+    print(req_url)
+    ensure_connection(req_url, 200)
+    resp = requests.get(req_url)
+    assert resp.status_code == 200, f"Expected 200 code for /metrics and got {resp.status_code}"
+    resp_content = resp.content.decode("utf-8")
+    for line in resp_content.splitlines():
+        if 'last_reload_milliseconds{class="%s"}' %ingress_class in line:
+            reload_metric = re.findall("\d+", line)[0]
```

Comment:
we can return here immediately, there is no need to search in other lines

Reply:
yep good catch 👍